src/TurbKineticEnergyEquationSystem.C

```diff
@@ -382,6 +390,13 @@ TurbKineticEnergyEquationSystem::register_inflow_bc(
   std::vector<double> userSpec(1);
   userSpec[0] = tke.turbKinEnergy_;
 
+  std::printf("Open file for writing tkeFS\n");
```

Comment:
Same with this one

Reply:
Those 2 write statements are a bit of a hack to get the inflow values for k & omega down to the source terms for k & omega. So they are written to a file when the inflow_bc is registered (TurbKineticEnergyEquationSystem.C) and then the file is opened up and read in the node kernel (node_kernels/TKESSTBLTM2015NodeKernel.C). There is probably a more direct way to accomplish. I am open to suggestions.

Reply:
I don't see any easy way to access the inflow boundary options in the nodeKernel, but perhaps this is better suited as a `turbulence_model_constant`?  I would think you could set the value when you register the inflow bdry so as not to require the user to enter these quantities in two places.  I think that is probably safer than trying to write and then open/read a file in the working directory.  @gantech @psakievich @marchdf let us know if you have any comments on if this is necessary/reasonable approach?

Reply:
> >One other thought on this, it is feasible (probably not advisable) for a user to have two different inflows with two different tkeFS and sdrFS values. This looks like it would take last read as the value that gets passed everywhere. Is that the intent? Perhaps these free stream values shouldn't be on the bc data if there is only intended to be 1 true value for the entire domain.>>

@psakievich I was only thinking of the case where you have one principle free stream. Yes, you could have a situation where you have, for example, one free stream from the west and one from the south. In that case, you would need bookkeeping for two values of tkeFS and sdrFS. I will attempt to save off the free stream values and add them to the Realm object when the inflow BCs are initialized.

Reply:
Transfer of freestream inflow values to  node_kernels via file write/read has been replaced by adding them to Realm class.

Transition model floats are now defined using constexpr.